app/apps/server/orchestrator.js

```diff
@@ -122,7 +123,48 @@ class AppServerOrchestrator {
 			return Promise.resolve();
 		}
 
-		return this._manager.updateAppsMarketplaceInfo(apps);
+		return this._manager.updateAppsMarketplaceInfo(apps)
+			.then((apps) => {
+				this.notifyAdminsAboutInvalidAppsIfNecessary(apps);
+				return apps;
+			});
+	}
+
+	notifyAdminsAboutInvalidAppsIfNecessary(apps) {
+		const invalidApps = apps.filter((app) => app.getLatestLicenseValidationResult().hasErrors());
+
+		if (invalidApps.length === 0) {
+			return;
+		}
+
+		const id = 'someAppInInvalidState';
+		const title = 'Warning';
+		const text = 'There is one or more apps in an invalid state. Click here to review.';
```

Comment:
should we use??
```suggestion
		const text = 'There_is_one_or_more_apps_in_an_invalid_state_Click_here_to_review';
```

Reply:
Why is this resolved? Does it not apply?